wp-content/themes/pub/wporg-learn-2020/archive-lesson-plan.php

```diff
@@ -15,6 +15,174 @@
+				<div class="lp-taxonomy-description"><?php echo esc_html__( "Browse lesson plans by the audience they're intended for.", 'wporg-learn' ); ?></div>
+				<?php foreach ( $audiences as $audience ) :
+					$is_sticky = get_term_meta( $audience->term_id, 'sticky', true );
+					if ( $is_sticky ) :
+						?>
+				<div class="card">
+					<a href="<?php echo esc_url( get_term_link( $audience ) ); ?>">
+						<div class="icon">
+							<?php
+							$audience_icon = get_term_meta( $audience->term_id, 'dashicon-class', true ) ?? 'wordpress-alt';
+							?>
+							<span class="dashicons dashicons-<?php echo esc_attr( $audience_icon ); ?>"></span>
+						</div>
+						<p class="taxonomy-title"><?php echo esc_html( $audience->name ); ?></p>
+					</a>
+				</div>
+						<?php
+				endif;
+			endforeach; ?>
+			</div>
+		</div>
+
+		<div class="row lp-two-col between">
+			<div class="lp-taxonomy">
+				<h2 class="h4 lp-taxonomy-header"><?php echo esc_html__( 'Level', 'wporg-learn' ); ?></h2>
+				<div class="lp-taxonomy-description"><?php echo esc_html__( 'What experience partipants need.', 'wporg-learn' ); ?></div>
+				<ul class="lp-two-col-list">
+					<?php
+					$levels = get_terms( 'level', array(
+						'hide_empty' => false,
+						'orderby'    => 'id',
+					) );
+
+					foreach ( $levels as $level ) : ?>
+					<li><a href="<?php echo esc_url( get_post_type_archive_link( 'lesson-plan' ) ) . '?level[]=' . esc_html( $level->term_id ); ?>"><?php echo esc_html( $level->name ); ?><span class="dashicons dashicons-arrow-right-alt2"></span></a></li>
+						<?php
+					endforeach;
+					?>
+				</ul>
+			</div>
+
+			<div class="lp-duration">
+				<h2 class="h4 lp-taxonomy-header"><?php echo esc_html__( 'Duration', 'wporg-learn' ); ?></h2>
+				<div class="lp-taxonomy-description"><?php echo esc_html__( 'How long a lesson is estimated to take.', 'wporg-learn' ); ?></div>
+				<ul class="lp-two-col-list">
+					<?php
+					$durations = get_terms( 'duration', array(
+						'hide_empty' => false,
+						'orderby'    => 'name',
+						'order'      => 'ASC',
+					) );
+
+					$duration_index = 0;
+					$any_duration_arr = array();
+					$lp_archive_url = get_post_type_archive_link( 'lesson-plan' );
+					foreach ( $durations as $duration ) :
+						if ( $duration_index < 3 ) :
+							?>
+					<li><a href="<?php echo esc_url( add_query_arg( array( 'duration[]' => $duration->term_id ), $lp_archive_url ) ); ?>"><?php echo esc_html( $duration->name ); ?><span class="dashicons dashicons-arrow-right-alt2"></span></a></li>
+							<?php
+						else :
+							$any_duration_arr['duration'][] = $duration->term_id;
+						endif;
+						$duration_index++;
+					endforeach;
+
+					if ( $any_duration_arr['duration'] ) :
+						?>
+						<li><a href="<?php echo esc_url( add_query_arg( $any_duration_arr, $lp_archive_url ) ); ?>"><?php echo esc_html__( 'Any(over 60m)', 'wporg-learn' ); ?><span class="dashicons dashicons-arrow-right-alt2"></span></a></li>
```

Comment:
There should be a space here
```suggestion
						<li><a href="<?php echo esc_url( add_query_arg( $any_duration_arr, $lp_archive_url ) ); ?>"><?php echo esc_html__( 'Any (over 60m)', 'wporg-learn' ); ?><span class="dashicons dashicons-arrow-right-alt2"></span></a></li>
```